CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Release built: _not released yet_
   - `/state/entity/page/non-fungibles/` (when using `explicit_metadata` opt-in) 
   - `/state/entity/page/fungibles/` (when using `explicit_metadata` opt-in)
 - Added missing `total_count` property to `/state/validators/list` response.
+- Fix `/transaction/account-deposit-pre-validation` for virtual, non persisted accounts. It no longer returns error with code 404 `Entity not found`. 
```

Comment:
The new terminology (for the last 6 months or so) has been `uninstantiated pre-allocated accounts`, as per https://docs.radixdlt.com/docs/account